fw/http.c

```diff
@@ -3624,6 +3624,43 @@ tfw_h2_req_set_loc_hdrs(TfwHttpReq *req)
 	return 0;
 }
 
+/** Fuse multiple cookie headers into one.
+ * Works only with TFW_STR_DUP strings. */
+static int
+write_merged_cookie_headers(TfwStr *hdr, TfwMsgIter *it)
+{
+	int r = 0;
+	static const DEFINE_TFW_STR(h_cookie, "cookie" S_DLM);
+	static const DEFINE_TFW_STR(val_dlm, "; ");
+	static const DEFINE_TFW_STR(crlf, S_CRLF);
+	TfwStr *dup, *dup_end;
+	const TfwStr *cookie_dlm = &h_cookie;
+
```

Comment:
Let's write `cookie` before the loop and we won't need change `cookie_dlm` value as well as the whole variable

Reply:
That would just make code a bit worse as you'll have to track if separator is needed to be written or not.